docs/azure/sdk/aspnetcore-guidance.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Best practices for using the Azure SDK with ASP.NET Core
+description: Learn best practices and the steps to properly implement the Azure SDK for .NET in your ASP.NET Core apps.
+ms.topic: conceptual
+ms.custom: devx-track-dotnet
+ms.date: 10/22/2024
+---
+
+# Use the Azure SDK for .NET in ASP.NET Core apps
+
+The Azure SDK for .NET enables ASP.NET Core apps to integrate with many different Azure services. In this article, you'll learn best practices and the steps to implement the Azure SDK for .NET in your ASP.NET Core apps. You'll learn how to:
```

Comment:
nit: "implement the Azure SDK for .NET" sounds like what we do on our team ... is there another turn of phrase that would communicate that the reader's goal is to use Azure SDK for .NET clients in an ASP.NET Core app?

Reply:
This is an interesting consideration. I think "implement an SDK in your code" is a fairly standard phrase, but we could also use terms such as "incorporate, integrate, adopt", etc. Integrate would probably be my next choice.

Reply:
Cool, I haven't heard it, but if it is common, I will update my brain :)

Integrate sounds good to my ears, but totally your call.